squad/manage.py

```diff
@@ -7,6 +7,10 @@ def main():
     if len(sys.argv) > 1 and sys.argv[1] == 'test':
         os.environ.setdefault("DJANGO_SETTINGS_MODULE", "test.settings")
         testing = True
+
+        # skip tests that would raise exception in sqlite
+        if os.getenv('DATABASE') is None or os.getenv('DATABASE') == '':
```

Comment:
you can set DATABASE to explicitly use sqlite. 

I suggest using a variable called, say, EXCLUDE_TAG, and use it's value. so this can be fully controlled from e.g. the travis config

Reply:
I wanted `./manage.py test` to run against postgres by default, since that's our production's. OTOH testing in sqlite3, like we also do is interesting so that SQUAD can have a "testing" db.

I understood `EXCLUDE_TAG`, let me work on that.